next.config.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-param-reassign */
+module.exports = {
+  webpack: (config, { isServer }) => {
+    // Fixes npm packages that depend on `fs` module
+    if (!isServer) {
+      config.node = {
+        fs: 'empty',
+      }
+    }
+
+    return config
+  },
+}
```

Comment:
I'm wondering if this is necessary since we are not interacting with `fs`.

Reply:
Yep, that should be removed because we already are not using dotenv anymore